tests/fixtures/sqlite_database.py

```diff
@@ -0,0 +1,33 @@
+from logging import Logger
+from typing import Callable
+from unittest import mock
+
+import pytest
+
+from invokeai.app.services.config.config_default import InvokeAIAppConfig
+from invokeai.app.services.image_files.image_files_base import ImageFileStorageBase
+from invokeai.app.services.shared.sqlite.sqlite_database import SqliteDatabase
+from invokeai.app.services.shared.sqlite_migrator.migrations.migration_1 import migration_1
+from invokeai.app.services.shared.sqlite_migrator.migrations.migration_2 import migration_2
+from invokeai.app.services.shared.sqlite_migrator.sqlite_migrator_impl import SQLiteMigrator
+
+CreateSqliteDatabaseFunction = Callable[[InvokeAIAppConfig, Logger], SqliteDatabase]
+
+
+@pytest.fixture
+def create_sqlite_database() -> CreateSqliteDatabaseFunction:
```

Comment:
Why pass this around as a fixture rather than just defining a function and importing it?

Reply:
Good question. I saw some examples of using pytest fixtures in this way, but they were using the pattern to compose fixtures. I'm apparently not composing anything here, so the pattern is only adding confusion. I'll remove it.